code/modules/religion/rites.dm

```diff
@@ -253,19 +253,90 @@
+	"... grant this servant power ...",
+	"... grant them temporary immortality ...")
+	invoke_msg = "... Grant them the power to become one with necromancy!!"
+	favor_cost = 2250
+/// the creature chosen for the rite
+	var/mob/living/lich_to_be
+/// the the typepath of the spell to gran
+	var/lichspell = /obj/effect/proc_holder/spell/targeted/lesserlichdom
+
+/datum/religion_rites/create_lesser_lich/perform_rite(mob/living/user, atom/religious_tool)
+	if(!ismovable(religious_tool))
+		to_chat(user,"<span class='warning'>This rite requires a religious device that individuals can be buckled to.</span>")
+		return FALSE
+	var/atom/movable/movable_reltool = religious_tool
+	if(length(movable_reltool.buckled_mobs))
+		for(var/creature in movable_reltool.buckled_mobs)
+			lich_to_be = creature
+		if(!lich_to_be.mind.hasSoul)
+			to_chat(user,"<span class='warning'>[lich_to_be] has no soul, as such this rite would not help them. To empower another, they must be buckled to [movable_reltool].</span>")
+			lich_to_be = null
+			return FALSE
+		for(var/obj/effect/proc_holder/spell/knownspell in lich_to_be.mob_spell_list)
+			if(knownspell.type == lichspell)
+				to_chat(user,"<span class='warning'>You've already empowered [lich_to_be], get them to use the spell granted to them! To empower another, they must be buckled to [movable_reltool].</span>")
+				lich_to_be = null
+				return FALSE
+		to_chat(user,"<span class='warning'>You're going to empower the [lich_to_be] who is buckled on [movable_reltool].</span>")
+		return ..()
+	else
+		if(!movable_reltool.can_buckle) //yes, if you have somehow managed to have someone buckled to something that now cannot buckle, we will still let you perform the rite!
+			to_chat(user,"<span class='warning'>This rite requires a religious device that individuals can be buckled to.</span>")
+			return FALSE
+		lich_to_be = user
+		if(!lich_to_be.mind.hasSoul)
+			to_chat(user,"<span class='warning'>You have no soul, as such this rite would not help you. To empower another, they must be buckled to [movable_reltool].</span>")
+			lich_to_be = null
+			return FALSE
+		for(var/obj/effect/proc_holder/spell/knownspell in lich_to_be.mob_spell_list)
+			if(knownspell.type == lichspell)
+				to_chat(user,"<span class='warning'>You've already empowered yourself, use the spell granted to you! To empower another, they must be buckled to [movable_reltool].</span>")
+				lich_to_be = null
+				return FALSE
+		to_chat(user,"<span class='warning'>You're empowering yourself!</span>")
+		return ..()
+
+
+/datum/religion_rites/create_lesser_lich/invoke_effect(mob/living/user, atom/movable/religious_tool)
+	..()
+	if(!ismovable(religious_tool))
+		CRASH("[name]'s perform_rite had a movable atom that has somehow turned into a non-movable!")
+	var/atom/movable/movable_reltool = religious_tool
+	if(!length(movable_reltool.buckled_mobs))
+		lich_to_be = user
+	else
+		for(var/buckled in movable_reltool.buckled_mobs)
```

Comment:
unaddressed still

Reply:
ah, it was a hidden reply for some reason so I missed it